Make exception tracebacks default

Instead of asking the user to run coala with `-L DEBUG`

```
E       AssertionError: '' != '[DEBUG][18:36:51] 4 r34l ch4ll3n63: 123 Ä[35 chars]n→ d'
E       + [DEBUG][18:36:51] 4 r34l ch4ll3n63: 123 ÄÖü ABc @€¥ §&% {[( ←↓→↑ ĦŊħ ß°^ \
E       + → d

tests/output/printers/LogPrinterTest.py:60: AssertionError

```

--- a/coalib/output/printers/LogPrinter.py
+++ b/coalib/output/printers/LogPrinter.py
@@ -115,7 +115,7 @@ class LogPrinter:
 
         self.log(log_level, message, timestamp=timestamp, **kwargs)
         self.log_message(
-            LogMessage(LOG_LEVEL.DEBUG,
+            LogMessage(LOG_LEVEL.INFO,
                        "Exception was:" + "\n" + traceback_str,
                        timestamp=timestamp),
             **kwargs)

--- a/tests/output/printers/LogPrinterTest.py
+++ b/tests/output/printers/LogPrinterTest.py
@@ -112,7 +112,7 @@ class LogPrinterTest(unittest.TestCase):
         self.assertTrue(uut.printer.string.startswith(
             "[ERROR][" + self.timestamp.strftime("%X") +
             "] Something failed.\n" +
-            "[DEBUG][" + self.timestamp.strftime("%X") +
+            "[INFO][" + self.timestamp.strftime("%X") +
             "] Exception was:"))
 
         uut.log_level = LOG_LEVEL.INFO

